### ISSUE TITLE
Screen readers cannot detect word wrap

### ISSUE THREAD

**Issue by hussmail** [Microsoft/vscode#65625]
Issue Type: <b>Bug</b>

When I turn on word wrap and press down/up arrows the cursor does not move to the same column of the next/previous screen line but moves to the same column on the next/previous hard line. I tried this with NVDA and JAWS for Windows screen readers and the behavior is the same - the screen reader does not detect screen lines.

VS Code version: Code 1.30.1 (dea8705087adb1b5e5ae1d9123278e178656186a, 2018-12-18T18:12:07.165Z)
OS version: Windows_NT x64 10.0.17763

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM)2 Duo CPU     E8300  @ 2.83GHz (2 x 2826)|
|GPU Status|2d_canvas: unavailable_software<br>checker_imaging: disabled_off<br>flash_3d: unavailable_software<br>flash_stage3d: unavailable_software<br>flash_stage3d_baseline: unavailable_software<br>gpu_compositing: unavailable_software<br>multiple_raster_threads: disabled_off<br>native_gpu_memory_buffers: disabled_software<br>rasterization: unavailable_software<br>video_decode: unavailable_software<br>video_encode: unavailable_software<br>webgl: enabled_readback<br>webgl2: unavailable_off|
|Memory (System)|7.71GB (3.59GB free)|
|Process Argv||
|Screen Reader|yes|
|VM|50%|

</details>Extensions: none

**Comment by alexandrudima** [comment]
@hussmail When a screen reader is attached, we disable word wrapping specifically to workaround the problem that we cannot communicate wrapping points to screen readers. Please make sure to use `"editor.accessibilitySupport": "on"` in your user settings.

Are you perhaps configuring `"editor.accessibilitySupport": "off"` ?

**Comment by hussmail** [comment]
AccessibilitySupport was set to "Auto" but I changed it to "on" and nothing happened. Now my user setting is
{
    "editor.accessibilitySupport": "on"
}.
I tried to change word wrap by pressing alt+z and by selecting it from command pallet but nothing happened and the screen reader didn't tell me anything. So I changed word wrap to "on" by changing editor settings in editor settings window and it is set to "on" but the screen lines still not detected.
I think the screen reader should inform the user for changes in word wrap status.
